tests/flow/test_params.py

```diff
@@ -107,3 +107,36 @@ def test_id_scan(self):
         plan = redis_graph.execution_plan(query)
         self.env.assertIn('NodeByIdSeek', plan)
 
+    def test_invalid_cypher_options(self):
```

Comment:
I don't think this is the right place for these tests, don't we have query error / validation test?